### PR TITLE
Fix/devise forms

### PR DESCRIPTION
**Pull request by leocsdev**
This PR fixes the basic layout of devise forms for users as well as its errors and flash message notifications.

Sign up

img01-sign-up

Login

img02-login

Sign up errors

img03-signup-error

Login errors

img04-login-error

Edit profile part 1

img05-edit-user

Edit profile part 2
img06-edit-user

### CODE REVIEW

File: app/assets/stylesheets/home.scss [hunk 1]
```diff
@@ -1,92 +1,90 @@
...
+      }
+
+      .other-user-username {
+        font-size: 0.8rem;
+        color: #777777;
+      }
     }
   }
-  
-
```
Comment (Eunicegenel) [inline app/assets/stylesheets/home.scss:92]: add newline

File: app/views/devise/sessions/new.html.erb [hunk 2]
```diff
@@ -1,28 +1,49 @@
...
-  
-    <div class="actions">
-      <%= f.submit "Log in" %>
     </div>
-  <% end %>
-  
-  <%= render "devise/shared/links" %>
-</div>
+  </div>
+</section>
```
Comment (Eunicegenel) [inline app/views/devise/sessions/new.html.erb]: add newline

File: app/views/devise/passwords/new.html.erb [hunk 3]
```diff
@@ -1,18 +1,42 @@
...
-      <%= f.submit "Send me reset password instructions" %>
-    </div>
-  <% end %>
-  
-  <%= render "devise/shared/links" %>
-</div>
+  </div>
+</section>
+
+
```
Comment (Eunicegenel) [inline app/views/devise/passwords/new.html.erb]: remove excess lines

**Review by Eunicegenel — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/home.scss (hunk 1 above)
Comment (leocsdev) [reply app/assets/stylesheets/home.scss:92]: image

**Review by villanuevajamesfvillanueva — APPROVED** [review]

**Review by Eunicegenel — APPROVED** [review]

**Pull request merged**